akd/src/tree_node.rs

```diff
@@ -243,6 +243,14 @@ impl TreeNode {
     pub(crate) async fn write_to_storage<S: Storage + Send + Sync>(
         &self,
         storage: &S,
+    ) -> Result<(), StorageError> {
+        self.save_to_storage(storage, false).await
+    }
+
+    async fn save_to_storage<S: Storage + Send + Sync>(
```

Comment:
Suggestion: `write_to_storage_impl` instead of `save_to_storage`? It's not quite clear what the difference is between `write_to_storage` and `save_to_storage`, but with the `impl` suffix it's clear that one is a wrapper of another.

Reply:
Updated names and added docs for the `impl` version.